enterprise/models.py

```diff
@@ -312,6 +312,13 @@ class Meta:
         on_delete=models.CASCADE
     )
 
+    disable_expiry_messaging_for_learner_credit = models.BooleanField(
+        verbose_name="Disable expiration messaging for learner credit",
+        default=False,
+        help_text=_("Automatically disabled. If unchecked, learners and admins will not receive email, "
+                    "and modal notifications on expiring learner credit plans.")
```

Comment:
nit: maybe remove "modal" since it's both modals and alert banners? By not including the exact UI element (e.g., modal) here, it means the expiration experience can freely change its UI elements (e.g., if the expiration experience eventually doesn't have a modal one day).